openslides/core/urls.py

```diff
@@ -7,10 +7,15 @@
     url(r'^core/url_patterns/$',
         views.UrlPatternsView.as_view(),
         name='core_url_patterns'),
+
     url(r'^core/version/$',
         views.VersionView.as_view(),
         name='core_version'),
 
+    url(r'^angular_js/(?P<app>site|projector)/$',
```

Comment:
ups. I did not realize this. I like angular_js for. Because there could be other clients using javascript, but this file is only for the angular client.